src/containers/organizations/epics/CreateOrganizationEpic.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * @flow
+ */
+
+import {
+  DataModels,
+  Types,
+  OrganizationsApi,
+  PermissionsApi
+} from 'loom-data';
+
+import {
+  Observable
+} from 'rxjs';
+
+import * as OrgsActionTypes from '../actions/OrganizationsActionTypes';
+
+import {
+  AUTHENTICATED_USER
+} from '../../../utils/Consts/UserRoleConsts';
+
+const {
+  Acl,
+  AclBuilder,
+  AclData,
+  AclDataBuilder,
+  Ace,
+  AceBuilder,
+  Principal,
+  PrincipalBuilder
+} = DataModels;
+
+const {
+  ActionTypes,
+  PermissionTypes,
+  PrincipalTypes
+} = Types;
+
+function setNewOrgPermissions(newOrgId :UUID) {
```

Comment:
This should be moved into `constainers/permissions/PermissionEpic.js` (doesn't exist yet, feel free to create) as it's own epic: 
```
updatePermissionsEpic(actions$) {
  actions$.ofType(actionTypes.UPDATE_ACL)
  ...
}
```

and `createNewOrg` should put an `UPDATE_ACL` request in the observable. See `CatalogEpic.searchCatalog` for an example